test/unit/app.js

```diff
@@ -1427,7 +1420,7 @@ describe('Unit tests for swiftserver:app', function () {
           appType: 'scaffold',
           appName: applicationName,
           capabilities: [],
-          services: { watsonconversation: {} }
+          services: { conversation: {} }
```

Comment:
doesn't actually test the key `conversation` exists unless we put something real in the value (not the empty object)